src/system/UsersModule/Entity/Repository/UserRepository.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2015 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ * @package Zikula_Form
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\UsersModule\Entity\Repository;
+
+use Doctrine\ORM\EntityRepository;
+use Zikula\UsersModule\Entity\UserEntity;
+use Zikula\UsersModule\Entity\RepositoryInterface\UserRepositoryInterface;
+
+class UserRepository extends EntityRepository implements UserRepositoryInterface
+{
+    public function findByUids($uids)
```

Comment:
Yes, simply renaming to `byUids`. If Doctrine would not have been used, the entities might not be "found" but maybe "generated" or "streamed" or "read". "Finding" is a bit more Doctrine and database specific.

Don't get me wrong here. I'm perfectly fine with the current naming. Chances that Doctrine isn't used are rather small anyways. Just wanted to present the option to rename the method.